.travis.yml

```diff
@@ -1,24 +1,56 @@
 language: python
-
 python:
-  - '2.7'
-  - '3.6'
-
-cache: pip
+    - "2.7"
+    - "3.6"
 
 services:
-  - mysql
+    - mysql
 
+# command to install dependencies
+# pip locked to version in P_deliver
```

Comment:
... locked to P_main?

Reply:
P_deliver is not right, I'll change that, where does it run on python 2.7?

Reply:
Looks like nowhere. it's python 3.7 for hasta (in P_demux) and python 3.6 for thalamus (in prod)

Reply:
ok, let's change it to prod then and change 2.7 to 3.7 :)